gnome-initial-setup/pages/password/gis-password-page.c

```diff
@@ -206,6 +206,26 @@ validate (GisPasswordPage *page)
   return FALSE;
 }
 
+static void
+page_visibility_changed (GtkWidget  *w,
+                         GParamSpec *pspec,
+                         gpointer   *unused)
```

Comment:
Sorry for the nitpicking, but this should be `gpointer` (`gpointer *` makes it a double pointer)

Reply:
WTF... this is what happens when I have my brain debugging things on our insane apps grid I guess...